### DISCUSSION TITLE
Switch node with Collect Hit Strategy

### DISCUSSION THREAD

**Discussion by prasunsrivastava** [gorules/zen#191] [Q&A]
Hi Everyone, 

I have a switch node where one of the conditions branches out to 3 different flows. I use Collect Hit Strategy but it does not branch out to these 3 paths if the condition matches. Am I doing something wrong?

Thanks

**Comment by ivanmiletic** [comment]
Hi, would you be able to provide example? Switch node with collect hit should branch to all branches where expression resolves to true
